kolibri/plugins/learn/assets/src/views/exam-list/index.vue

```diff
@@ -7,27 +7,39 @@
       <page-header :title="$tr('examName')"></page-header>
       <p v-if="activeExams" class="exams-assigned">{{ $tr('assignedTo', { assigned: activeExams }) }}</p>
       <p v-else class="exams-assigned">{{ $tr('noExams') }}</p>
-      <div class="exam-row" v-for="exam in exams" :key="exam.id">
-        <mat-svg class="exam-icon" slot="content-icon" category="action" name="assignment_late"/>
-        <h2 class="exam-title">{{ exam.title }}</h2>
-        <div class="exam-details" v-if="exam.closed || !exam.active">
-          <p class="answer-count">{{ $tr('howManyCorrect', { score: exam.score, outOf: exam.questionCount })}}</p>
-          <div class="button-or-score">
-            <b>{{ $tr('percentCorrect', { pct: exam.score/exam.questionCount })}}</b>
-          </div>
+
+      <div class="pure-g exam-row" v-for="exam in exams" :key="exam.id">
+
+        <div class="exam-row-1st-col" :class="firstColClass">
+          <mat-svg class="exam-icon" slot="content-icon" category="action" name="assignment_late"/>
+          <h2 class="exam-title">{{ exam.title }}</h2>
         </div>
-        <div class="exam-details" v-else>
-          <p class="answer-count" v-if="exam.answerCount !== null">
-            {{ $tr('questionsLeft', { left: exam.questionCount - exam.answerCount }) }}
-          </p>
-          <div class="button-or-score">
+
+        <template v-if="exam.closed || !exam.active">
+          <div class="exam-row-2nd-col" :class="secondColClass">
```

Comment:
When in 'mobile', recommend that the score column be left-aligned.

(One strategy for `:class` would be return an array of classes rather than a single one, and then conditionally append them depending on breakpoint.)